### ISSUE TITLE
Disable Butterick by default

### ISSUE THREAD

**Issue by ndarville** [amperser/proselint#393]
The presumptive use of Proselint is for unrendered plaintext (Markdown, RTF, TXT). In light of this, I can’t make sense of using Butterick as a default filter.

The [documentation for Markdown basics][docs] uses non-curly quotes and triple-period ellipses, whereas the surrounding, post-processed text, uses the “proper versions”, since it have been parsed by SmartyPants.

Given the dedication to thwarting false positives, this feels like an extreme case of a false positives that might deter people from using Proselint after running it the first time on their plaintext files. Besides the detriment to the experiment, it doesn’t makes sense. if we assume some basic usage cage for the linter. :)

[docs]: http://daringfireball.net/projects/markdown/basics

**Comment by suchow** [comment]
Agreed. What I'd prefer to do is to have Proselint detect the markup, either using the file extension or some other means, and then suppress any checks that will false alarm on the markup.

See also the LaTeX version of this issue, #277.

**Comment by gvoysey** [comment]
Just ran into this.  How do i shut this warning up?

**Comment by dialex** [comment]
I tried to disable it using proselint's ignore file. Did not work. In this screenshot you see the error, the original file with "proper" quotes, that weird env var, and my `config` (proselint ignore) file.

<img width="1162" alt="Screenshot 2021-05-29 at 14 27 18" src="https://user-images.githubusercontent.com/2140450/120072168-4aed0800-c08a-11eb-8978-0d4f033ade26.png">

**Comment by Nytelife26** [comment]
This will be fixed in a future release, fortunately. We're working on our configuration system currently.
I'm not sure when our release cycles will begin again, but I'll keep you all informed. I apologise for the inconvenience and thank you greatly for sticking with proselint, even after the three years with no maintenance.
